src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -291,6 +289,7 @@ function MdAutocomplete ($$mdSvgRegistry) {
           itemTemplate    = getItemTemplate(),
           leftover        = element.html(),
           tabindex        = attr.tabindex;
+      var menuContainerClass = attr.mdMenuContainerClass ? ' ' + attr.mdMenuContainerClass : '';
```

Comment:
This is not interpreting bindings, isnt it? O.o :(

Reply:
Nope, like `md-menu-class`, it only accepts a `string` and doesn't apply a watcher.